fetching/strat.py

```diff
@@ -12,9 +13,14 @@ class ApplicationStrategy(ABC):
     functions to be traced upon execution.
     """
 
+    def __init__(self, recurse_into_subdirs: bool = True):
+        self.globber = pathlib.Path.rglob if recurse_into_subdirs else pathlib.Path.glob
```

Comment:
Rename this to get_files_in_path_function or something similar.

Reply:
I'd rather not have such a long name

Reply:
Then get_files_in_path? Because globber doesn't tell me anything.

Reply:
`globber` matches the naming of the `rglob` and `glob` methods.

Reply:
Sure, we can rename it. But the mechanic of globbing should be recognisable from its